### DISCUSSION TITLE
Question:  Kadalu Integration with Kubernetes 1.28 or later?

### DISCUSSION THREAD

**Discussion by amir-iqbal-cg** [kadalu/kadalu#1019] [Q&A] [answered]
Hi Everyone, 
     We are using K8 (v.1.20.X) and planning to upgrade it to 1.28.xx version. 
      Currently we have integrated Kadalu (v.0.7)  therefore i am interested to know if Kadalu will continue to work for K8  v.1.28 and higher? Since GlusterFS is officially no more supported by K8 Community. 

Looking forward hearing from you. Thanks

BR. 
Amir

**Comment by amir-iqbal-cg** [comment]
bump

**Comment by amir-iqbal-cg** [comment]
apparently, this community is dead. sad :(

**Comment by aravindavk** [comment, marked as answer]
Kubernetes deprecated the intree Gluster plugin(https://groups.google.com/g/kubernetes-dev/c/-50-3MVw4Ck). Kadalu is not dependent on that plugin so not impacted by that decision.

**Comment by amir-iqbal-cg** [comment]
appreciate your feedback aravindavk.  That's what i was looking for 👍
